api/libs/marxan-output/src/marxan-execution-metadata.geo.entity.ts

```diff
@@ -32,4 +38,7 @@ export class MarxanExecutionMetadataGeoEntity {
     name: `output_zip`,
   })
   outputZip?: Buffer | null;
+
+  @CreateDateColumn({ type: 'timestamp without time zone' })
```

Comment:
can we please stick to `snake_case` for column names, even at the expense of explicitly setting `name: 'created_at'`?

Reply:
good catch, thanks!